CHANGELOG.md

```diff
@@ -8,12 +8,25 @@
   `Option<Vec<u8>>`.
 - `ReadonlyStorage::range` and all its implementations now return an iterator
   over `Option<KV>` instead of `Option<StdResult<KV>>`.
+- `Storage::{set, remove}` and all their implementations no longer have a return value.
+  Previously they returned `StdResult<()>`.
 - Trait `Querier` is not `Clone` and `Send` anymore.
+- `consume_region` panics on null pointers and returns `Vec<u8>` instead of
+  `StdResult<Vec<u8>>`.
+- Added contract migration mechanism. Contracts are now also expected to expose a `migrate`
+  function with the following definition:
```

Comment:
"are now also expected to expose" -> "can now optionally export a"

Having such a function only makes sense if the contract is an upgrade of a different contract. I don't know how it is implemented right now but we should only call it when present.

Reply:
Ahhh, i see.
Corrected in next commit